pkg/fuzzer/job.go

```diff
@@ -98,9 +99,16 @@ func (job *triageJob) run(fuzzer *Fuzzer) {
 	if stop {
 		return
 	}
+	var wg sync.WaitGroup
```

Comment:
Do we need to wait for them? Can this be just this:
```
	for call, info := range job.calls {
		go job.handleCall(call, info)
	}
```
?

Reply:
I think we do want to wait for them. We display the number of running triage jobs and want it to be correct + we use the number of running candidate triage jobs to decide when to start hub querying / bug reproductions in syz-manager.